source/pay-product-page/javascripts/cookie-banner.js

```diff
@@ -0,0 +1,253 @@
+// import InitialiseAnalytics from '../analytics/init'
```

Comment:
I will look at refactoring this file - the styling of the JS.

Reply:
I can remove this for now actually... analytics will go into another PR.

Reply:
I meant to say the whole file - use 1 style of JS. 
We can take a look at it.